website/project/views/contributor.py

```diff
@@ -462,7 +462,7 @@ def send_claim_email(email, user, node, notify=True, throttle=24 * 3600):
 
 
 @contributor_added.connect
-def notify_added_contributor(node, contributor, auth=None, throttle=None):
+def notify_added_contributor(node, contributor, auth=None, throttle=None, all_global_subscriptions_none=None):
```

Comment:
`all_global_subscriptions_none` appears to be used as a boolean, so it should default to `False` instead of `None`